### PR TITLE
A11-2357 :wrench: fix Menu ClickableText svg size

### PR DESCRIPTION
**Pull request by charbelrami**
## Context

After some tests, I believe this change would fix [A11-2357 : Fix regressions to clickable text](https://linear.app/noredink/issue/A11-2357/fix-regressions-to-clickable-text)

### Before
localhost_8000_ (1)

### After
localhost_8000_

@NoRedInk/design 

## Component completion checklist

- [x] I've gone through the relevant sections of the [Development Accessibility guide](https://paper.dropbox.com/doc/Accessibility-guide-4-Development--BiIVdijSaoijjOuhz3iTCJJ1Ag-rGoHpC91pFg3zTrYpvOCQ) with the changes I made to this component in mind
- [x] Changes are clearly documented
    - [x] Component docs include a changelog
    - [x] Any new exposed functions or properties have docs
- [x] Changes extend to the Component Catalog
    - [x] The Component Catalog is updated to use the newest version, if appropriate
    - [x] The Component Catalog example version number is updated, if appropriate
    - [x] Any new customizations are available from the Component Catalog
    - [x] The component example still has:
        - an accurate preview
        - valid sample code
        - correct keyboard behavior
        - correct and comprehensive guidance around how to use the component
- [x] Changes to the component are tested/the new version of the component is tested
- [x] Component API follows standard patterns in noredink-ui
    - e.g., as a dev, I can conveniently add an `nriDescription`
    - and adding a new feature to the component will _not_ require major API changes to the comopnent

### CODE REVIEW

File: src/Nri/Ui/Menu/V4.elm [hunk 1]
```diff
@@ -955,6 +956,15 @@ viewClickableText title menuConfig clickableTextAttributes attributes =
         ([ ClickableText.custom attributes
          , ClickableText.disabled menuConfig.isDisabled
          , ClickableText.rightIcon (AnimatedIcon.arrowDownUp menuConfig.isOpen)
+         , ClickableText.css
+            [ Css.Global.descendants
+                [ Css.Global.selector "svg"
```
Comment (tesk9) [inline src/Nri/Ui/Menu/V4.elm:961]: I think this will resize the left icon too, if the user adds one, right?

Comment (charbelrami) [reply src/Nri/Ui/Menu/V4.elm]: right! Thanks! Made some changes

File: src/Nri/Ui/ClickableText/V3.elm [hunk 2]
```diff
@@ -394,6 +394,22 @@ appearsInline =
...
+        (\config ->
+            { config
+                | rightIconStyles =
+                    List.append config.rightIconStyles
+                        [ Css.width (Css.px 15)
+                        , Css.height (Css.px 15)
+                        , Css.marginLeft (Css.px 8)
+                        ]
+            }
+        )
```
Comment (tesk9) [inline src/Nri/Ui/ClickableText/V3.elm:410]: I like the rightIconStyles idea!

What do you think about exposing `rightIconCss : List Css.Style -> Attribute msg` instead, and have the Menu component do the styling?

Comment (charbelrami) [reply src/Nri/Ui/ClickableText/V3.elm]: I like it! Updated the code!

**Comment by tesk9** [comment]
Looks great to me! Could you please rebase or merge master to get https://github.com/NoRedInk/noredink-ui/pull/1306 in to hopefully fix Ci, and then I think this is ready to gooo!

**Pull request merged**